unlock-js/src/constants.js

```diff
@@ -4,6 +4,7 @@ export const GAS_AMOUNTS = {
   purchaseKey: 300000,
   withdrawFromLock: 1000000,
   partialWithdrawFromLock: 1000000,
+  deployContract: 4500000,
```

Comment:
Ah so it was deploy you were talking about.  I think this limit is aggressive and likely to break again in the future.  I'd recommend maybe 6mil instead, which is close to the network max of approx 6.4mil.  This is not a call we wish to optimize in any way, so no need to set an aggressive max.

Reply:
great thank you